app/views/handlers/list_add_question.py

```diff
@@ -18,7 +37,7 @@ def get_next_location_url(self):
 
     def handle_post(self):
         # Ensure the section is in progress when user adds an item
-        list_item_id = self.questionnaire_store_updater.add_list_item(
+        self._list_item_id = self.questionnaire_store_updater.add_list_item(
```

Comment:
I am not sure about reassigning the value of self._list_item_id, can we keep it as it was?

Reply:
We need an attribute to capture the generated list item id as it is needed in the `get_next_location_url method`